lib/src/maintenance.dart

```diff
@@ -358,20 +361,21 @@ Future<Maintenance> detectMaintenance(
           .join();
       sb.write(items);
 
-      final level = (errorCount > 0 || warningCount > 0)
+      final level = (bulkErrorCount > 0 || bulkWarningCount > 0)
           ? SuggestionLevel.warning
           : SuggestionLevel.hint;
       maintenanceSuggestions.add(
         new Suggestion(
           SuggestionCode.bulk,
           level,
-          'Fix analysis and formatting issues.',
+          'Fix additional ${restSuggestions.length} files with analysis or formatting issues.',
```

Comment:
Is restSuggestions.length always more than 1? Should pluralize be used here?

Reply:
Yes, it is always 3+, because it starts to work when the total is >= 5, and 2 of those will be taken out.